iris/__init__.py

```diff
@@ -5,6 +5,9 @@
 import os
 import sys
 import webbrowser
+import shutil
```

Comment:
Sort imports in alphabetical order. 

`ruff` linter checks will capture this and should be enabled for the project. See https://github.com/gt-sse-center/RepoAuditor/blob/aa64ee314688d7465436bf6da083afba8b3703e0/pyproject.toml#L75 for configuration info.